test/TestConfig/scripts/testKitGen/testKitGen.pl

```diff
@@ -79,9 +82,9 @@
 			. "                            If the modesXml is not provided, the program will try to find modes.xml under projectRootDir/TestConfig/resources.\n"
 			. "  --ottawaCsv=<path>        Path to ottawa.csv file.\n"
 			. "                            If the ottawaCsv is not provided, the program will try to find ottawa.csv under projectRootDir/TestConfig/resources.\n"
-			. "  --buildList=<paths>       Comma separated project paths to search playlist.xml. The paths are relative to projectRootDir.\n"
-			. "                            If the buildList is not provided, the program will search all files under projectRootDir.\n";
-		die "Please specify valid options!";
+			. "  --iterations=<number>     The iterations to repeat for each generated test command.\n"
```

Comment:
Repeatedly generate test command based on iteration number. The default value is 1.

Reply:
updated